src/session/utils.c

```diff
@@ -44,9 +46,22 @@ void _z_timestamp_reset(_z_timestamp_t *tstamp) {
     tstamp->_time = 0;
 }
 
+int8_t _z_session_generate_pid(_z_bytes_t *bs, uint8_t size) {
```

Comment:
The correct name should be `zid` and not `pid`.

Reply:
Done in the upcoming commits.